src/main/java/io/vavr/collection/Seq.java

```diff
@@ -1308,4 +1308,21 @@ default boolean startsWith(Iterable<? extends T> that, int offset) {
     default boolean isSequential() {
         return true;
     }
+
+    default <K, V> Map<K, V> toMap(Function<? super T, K> keyMapper, Function<? super T, ? extends V> valueMapper, Function<Seq<? extends V>, ? extends V> mergeFunction) {
+        Objects.requireNonNull(keyMapper, "keyMapper is null");
+        Objects.requireNonNull(valueMapper, "valueMapper is null");
+        Objects.requireNonNull(mergeFunction, "mergeFunction is null");
+
+        return this
```

Comment:
An alternative way to do this in Value could be:
```Java
static <T, K, V, E extends Tuple2<? extends K, ? extends V>, R extends Map<K, V>> R toMap(
            Value<T> value, R empty, Function<E, R> ofElement, Function<Iterable<E>, R> ofAll, Function<? super T, ? extends E> f, BiFunction<V,  V, V> mergeFunction) {
        if (value.isEmpty()) {
            return empty;
        } else if (value.isSingleValued()) {
            return ofElement.apply(f.apply(value.get()));
        } else {
            return Iterator.ofAll(value)
                    .map(f)
                    .groupBy(Tuple2::_1)
                    .mapValues(s -> s.map(Tuple2::_2).reduce(mergeFunction));
        }
    }
```
but makes the compiler a bit unhappy.